lib/topological_inventory/orchestrator/source.rb

```diff
@@ -55,10 +57,15 @@ def load_credentials(api)
         self.endpoint = api.get_endpoint(attributes['id'], tenant)
         return if endpoint.nil?
 
-        self.authentication = api.get_authentication(endpoint['id'], tenant)
-        return if authentication.nil?
+        if endpoint["receptor_node"].present?
+          self.authentication = AUTH_NOT_NECESSARY
```

Comment:
@lindgrenj6 Does it make sense to call it DELEGATE_AUTHENTICATION_TO_RECEPTOR instead of AUTH_NOT_NECESSARY? Since the receptor is the one that is storing the credentials or tokens which will be used when connecting to the Source.

Reply:
Another thing I noticed in the sources database is authtype: "receptor_node"

```
#<Authentication id: 10820, resource_type: "Endpoint", resource_id: 10898, name: nil, authtype: "receptor_node", 
username: nil, password: nil, status: nil, status_details: nil, tenant_id: 2, extra: nil, availability_status: nil, 
availability_status_error: nil, last_checked_at: nil, last_available_at: nil>
```

Reply:
https://github.com/RedHatInsights/sources-api/blob/69289f0f834f1b37f4d0d8c96ab5f09101967f7e/db/seeds/application_types.yml#L55

Reply:
Should we be adding another authtype here to be in sync with satellite
https://github.com/RedHatInsights/sources-api/blob/69289f0f834f1b37f4d0d8c96ab5f09101967f7e/db/seeds/application_types.yml#L10

Reply:
We probably will make orchestrator smarter in the future, but right now we're just trying to get the current iteration working with receptor sources. Right now Orchestrator doesn't even look at AuthTypes, we just grab authentications. 

I updated the logic to just check in the `digest` method since that was all that was breaking before, now we don't need a constant either.